scripts/check-mdc-tests-config.ts

```diff
@@ -185,5 +185,15 @@ export const config = {
       'should set the old snack bar animation state to complete and the new snack bar ' +
         'animation state to visible on entry of new snack bar',
     ],
+    'mdc-tabs': [
+      // These tests are excluded because they are verifying behavior that is not supported in MDC.
+      'should have no explicit roles',
```

Comment:
Should the feature or missing tests be ported over? The MDC tabs should have feature parity with the existing ones.

Reply:
This PR changes MDC tabs to not have feature parity with the existing tabs. In the existing tabs, the `[tabPanel]` input is optional. If it isn't provided, then the tabs fall back to the standard link/navigation behavior to avoid breaking existing instances in g3 that are impossible for me to migrate to use the `[tabPanel]`. In the new MDC tabs, we throw an error if the `[tabPanel]` input isn't provided. Thus, as users migrate from existing --> MDC, if they're not already using the `[tabPanel]` input, they'll need to do so.

Given the `[tabPanel]` input is enforced in MDC tabs, we don't need to verify/test any of the standard link/navigation fallback behavior. That's why I explicitly didn't port over these tests.